completed prep-css-syntax

Anthony-Pineda opened the pull request:
![Screenshot 2021-11-09 205231](https://user-images.githubusercontent.com/93688727/141052282-699a62f6-6493-41ff-aca0-1e79a399f230.jpg)

Anthony-Pineda committed:
completed prep-css-syntax

thebearingedge commented on prep-css-syntax/index.html:
```diff
@@ -0,0 +1,20 @@
+        p {font-size: 2rem;}
+        h3 {color: firebrick;}
+        h2{font-style: itlaic;}
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>CSS Syntax</title>
```
Be careful to only put CSS code inside of your `<style></style>` element.

Anthony-Pineda committed:
corrected style element placings

Anthony-Pineda commented:
![Screenshot 2021-11-10 173254](https://user-images.githubusercontent.com/93688727/141221292-6d0eacb3-80eb-411e-a893-13c0196b5afb.jpg)

thanks Tim, i have updated the placing of the style elements and fixed the italic typo
